packages/lib/src/components/ANCV/ANCV.tsx

```diff
@@ -99,9 +99,6 @@ export class ANCVElement extends UIElement<ANCVConfiguration> {
                 <CoreProvider i18n={this.props.i18n} loadingContext={this.props.loadingContext} resources={this.resources}>
                     <SRPanelProvider srPanel={this.props.modules.srPanel}>
                         <Await
-                            ref={ref => {
```

Comment:
Both QR loader and the Await component does not use this ref. The ref is usually used to hook into a component's `setStatus` and `showValidation` functions, both of which these components don't implement. This makes passing this ref unnecessary. If we ever need this component's to do so, we can add this later.